src/engine/Engine.cpp

```diff
@@ -47,6 +48,7 @@ Engine::Engine()
     , _verbosity( Options::get()->getInt( Options::VERBOSITY ) )
     , _lastNumVisitedStates( 0 )
     , _lastIterationWithProgress( 0 )
+    , _splittingStrategy( GlobalConfiguration::SPLITTING_HEURISTICS )
```

Comment:
So, recently we've been treating mutable configuration options by moving them from GlobalConfig to Options. That way you can set it at will, and also access it without passing it around. I suggest to do that here, as well.

Reply:
Moved to Options